test/JDBC/input/BABEL-EXTENDEDPROPERTY-v2-vu-cleanup.mix

```diff
@@ -0,0 +1,24 @@
+-- tsql
+USE babel_extended_properties_db
```

Comment:
Comments on the combined test cases files:

- Please add test cases for object/column (etc) names that use bracketed identifiers and double-quoted identifiers, as well as such identifiers containing non-standard characters (like space, comma, etc)

- All objects in the test cases seem to be created with lowercase names, please soem test caes where the object is created in mixed-case and the sp_XXXextendedproperty procs are called with a different case.

- Please add test cases where identifiers and attribute descriptions contains non-western characters (Chinese characters would be a good test case)

- Are there any negative tests where an identifier does not exist or the executing user has no permissions to access the object?

Reply:
1. done.
2. What you want is the test case for table column. Since table column has the most detailed object types, I add test case of trailing space, uppercase/lowercase. And I add test case for long name(more than 64) of shema, table, table column.
3. I have tested Chinese characters before, and you give the below answer. 
![image](https://github.com/babelfish-for-postgresql/babelfish_extensions/assets/5339358/7653a251-3bff-43ef-9dcc-e47d84233aab)
I think it's not the problem of my implementation.
4. Yes, all of negative test are considered, including not exist objects and permission check.
I create two users to test permissions, one is sysadmin, the other one is normal user.